extension/js/agent/utils/lazyWorker.js

```diff
@@ -3,10 +3,10 @@
   Agent.LazyWorker = Marionette.Object.extend({
 
     // time to wait until starting work
-    deferTime: 200,
+    deferTime: 100,
```

Comment:
had to bump this down because the heartbeat was 200ms. now we've got two heartbeats lol... cant wait for their cadence in the flame charts.